src/dsp_tools/commands/xmlupload/xmlupload.py

```diff
@@ -497,34 +498,39 @@ def _handle_upload_error(
     It then quits the Python interpreter with exit code 1.
 
     Args:
-        err_msg: string representation of the error that was the cause of the abort
+        err: the error that was the cause of the abort
         iri_resolver: a resolver for internal IDs to IRIs
         pending_resources: resources that were not uploaded to DSP
         failed_uploads: resources that caused an error when uploading to DSP
         pending_stash: an object that contains all stashed links that could not yet be reapplied to their resources
         config: the upload configuration
         permissions_lookup: dictionary that contains the permission name as string and the corresponding Python object
     """
-    logfiles = ", ".join([handler.baseFilename for handler in logger.handlers if isinstance(handler, FileHandler)])
-    msg = (
-        f"\n==========================================\n"
-        f"{datetime.now()}: xmlupload must be aborted because of an error.\n"
-        f"Error message: '{err_msg}'\n"
-        f"For more information, see the log file: {logfiles}\n"
-    )
+    if isinstance(err, XmlUploadInterruptedError):
+        msg = "\n==========================================\n" + err.message + "\n"
```

Comment:
I would not consider this an error, because the interruption is wanted. Therefore this message may not be optimal

Reply:
You mean the variable names are not optimal? I mean, because the message that is displayed to the user doesn't call it an error, it looks like this: 

```
==========================================
Interrupted: Maximum number of resources was reached (15)
Saved the current upload state to /Users/nussbaum/.dsp-tools/xmluploads/localhost/resumable/latest.pkl.
```

Reply:
I could consider renaming `XmlUploadInterruptedError` to `XmlUploadInterruptedException`, then it doesn't have the connotation of error.

Reply:
`xmlupload must be aborted because of an error` is not printed in case of an `XmlUploadInterruptedError`